mitchell_arnold/for_loop_basic1.py

```diff
@@ -0,0 +1,34 @@
+(1)  Basic 
+
+for i in range (0,151):
+	print(i)
+
+(2) Multiples of Five 
+
+for i in range (5, 1000000, 5):
+		print(i)
+
+(3)	Counting, the Dojo Way
+
+for i in range (1,101):
+	if (i % 5 == 0):
+		print("Coding")
+	if (i % 10 == 0):
+		print("Dojo")
```

Comment:
for numbers not divisible by 5 and 10, your function should print the number. This is a famous problem called Fizzbuzz (Please make sure you do it right)